### PR TITLE
Supports big screens mobile devices for popover menu

### PR DESCRIPTION
**Pull request by AlfredoAlc**
### Details

Changes the way it renders the popover menu for iPad, for the same as Web and Desktop.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5754

### Tests

1. Enter Expensify App with an iPad.
2. Press the FAB(Plus button).
3. The Popover menu should appear the way it appears in Web and Desktop.(Animated from the side instead of from bottom)

### QA Steps

1. Enter Expensify App with an iPad.
2. Press the FAB(Plus button).
3. The Popover menu should appear the way it appears in Web and Desktop.(Animated from the side instead of from bottom)

### Tested On

- [x] Web
- [ ] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Screen Shot 2021-11-10 at 17 06 28

#### Desktop

Screen Shot 2021-11-10 at 17 05 24

##### iPhone
iPhone

##### iPad

https://user-images.githubusercontent.com/53409604/141218476-427ada44-36fc-4323-9478-e8d716ea426d.mp4

#### Android

Android

### CODE REVIEW

**Review by johnmlee101 — DISMISSED** [review]
Looks good! Please resolve conflicts and I'll merge!

File: src/components/Popover/index.native.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const propTypes = {
  * This is a convenience wrapper around the Modal component for a responsive Popover.
  * On small screen widths, it uses BottomDocked modal type, and a Popover type on wide screen widths.
  */
+
```
Comment (johnmlee101) [inline src/components/Popover/index.native.js:16]: Why this change?

**Review by johnmlee101 — APPROVED** [review]

**Comment by johnmlee101** [comment]
Thanks for your patience!

**Comment by Jag96** [comment]
It looks like this PR resulted in a few deploy blockers:
- https://github.com/Expensify/App/issues/6520
- https://github.com/Expensify/App/issues/6519
- https://github.com/Expensify/App/issues/6522

Reverting this PR in https://github.com/Expensify/App/pull/6523 to unblock the deploy cc @AlfredoAlc @johnmlee101

**Comment by parasharrajat** [comment]
@AlfredoAlc the PR is reverted. Would you please resubmit it without the regression?

**Comment by AlfredoAlc** [comment]
Yes, I will be working on fixing those issues!
